app/views/users/two_factor_authentication_setup/index.html.erb

```diff
@@ -23,24 +23,46 @@
       <legend class="margin-bottom-2 usa-sr-only"><%= @presenter.intro %></legend>
       <% @presenter.options.each do |option| %>
         <div id="<%= "select_#{option.type}" %>" class="<%= option.html_class %>">
-          <%= radio_button_tag(
-                'two_factor_options_form[selection]',
-                option.type,
-                false,
-                disabled: option.disabled?,
-                class: 'usa-radio__input usa-radio__input--tile',
-              ) %>
-          <%= label_tag(
-                "two_factor_options_form_selection_#{option.type}",
-                class: 'usa-radio__label usa-radio__label--illustrated',
-              ) do %>
-                  <%= image_tag(asset_url("mfa-options/#{option.type}.svg"), alt: "#{option.label} icon", class: 'usa-radio__image') %>
-                  <div class="usa-radio__label--text tablet:padding-left-1"><%= option.label %>
-                    <span class="usa-radio__label-description">
-                      <%= option.info %>
-                    </span>
-                  </div>
-                <% end %>
+          <% if IdentityConfig.store.select_multiple_mfa_options %>
+            <%= check_box_tag(
+                  'two_factor_options_form[selection][]',
```

Comment:
I expect there might need to be some server-side changes to handle selection as an array, not a single option?

https://github.com/18F/identity-idp/blob/0ea9062aa8a61471ba388caf7cd41ef58d460dc5/app/forms/two_factor_options_form.rb#L16

Reply:
Should we bring back `self.selection = Array(paramss[:selection]).first`?

Reply:
@aduth that makes sense to change on the server side
@zachmargolis my thinking was to handle that first option in `mfa_redirect_helper.rb`. After seeing what helpers do, what I did may not be the best option and as it stands now. I can make your change and use it in a new controller?